script/__snapshots__/migrate-test-e2e.test.js.snap

```diff
@@ -0,0 +1,876 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`verify .all-contributorsrc 1`] = `
+"diff --git a/.all-contributorsrc b/.all-contributorsrc
+index 084b7ee..0ed629e 100644
+--- a/.all-contributorsrc
++++ b/.all-contributorsrc
+@@ -1,5 +1,5 @@
+ {
+-  \\"badgeTemplate\\": \\"<a href=\\\\\\"#contributors\\\\\\" target=\\\\\\"_blank\\\\\\"><img alt=\\\\\\"All Contributors: <%= contributors.length %> 🤝\\\\\\" src=\\\\\\"https://img.shields.io/badge/all_contributors-<%= contributors.length %>_🤝-21bb42.svg\\\\\\" /></a>\\",
```

Comment:
> `\\"`

Ah, this looks like it's on an old version of Vitest. This repo recently bumped to their latest major, which included a streamlining of snapshots (https://github.com/vitest-dev/vitest/pull/3961).

Running `pnpm run test:migrate -u` locally should update these.